lua/entities/gmod_wire_egp/lib/egplib/usefulfunctions.lua

```diff
@@ -444,8 +444,6 @@ function EGP:EGPCursor( this, ply )
 end
 
 function EGP.ScreenSpaceToObjectSpace(object, point)
-	point = { x = point.x - object.x, y = point.y - object.y }
```

Comment:
Won't this break other things? Translation has to also be accounted for, right?

Reply:
This function isn't called anywhere else but in `Contains` (which itself isn't called anywhere but in `egpObjectContainsPoint`). This avoids having to pass the EGP entity to the function in order to reorient itself to the global position using `GetGlobalPos`. An equivalent statement is done in `egpObjectContainsPoint` now.

Reply:
If it's not used anywhere else, can we just get rid of the functions? They aren't doing what their names imply now after your change.

Reply:
Works for me. Slipped in some other fixes, shouldn't be too disastrous.